tests/test_git_shell.py

```diff
@@ -1,6 +1,10 @@
+import os
+from unittest.mock import patch
+
 from ggshield.git_shell import GIT_PATH, is_git_dir, shell
 
 
+@patch.dict(os.environ, {"LANG": "C"})
```

Comment:
I tried doing it in `shell` by adding `["LANG", "C"]` before `["git", ...]`, but it didn't work. Plus, I'm not sure what the impact would be if we enforced the `LANG` everytime we call the git process (especially in CI environments).

Reply:
That is most likely because the `LANG=C git...` syntax is interpreted by your shell (as in, bash, zsh...). `shell()` uses `subprocess.run()` without going through the shell (which is a good thing) so you have to force the env with something like this:

```python
env = os.environ.copy()
env["LANG"] = "C"
subprocess.run(..., env=env)
```

I don't think it would be a problem on CI environments, where LANG is most likely to be set to C or en_US, but it could avoid issues when running on users machines, where things are more... exotic (I remember a bug in a tool parsing numbers which failed on Windows Thai machines because numbers were localized...)

Reply:
Ok I'll add that, thanks for your input!